addons/web/static/src/views/list/list_renderer.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="web.ListRenderer" owl="1">
         <div
-            class="o_list_renderer o_renderer table-responsive"
+            class="o_list_renderer o_renderer table-responsive vh-100"
```

Comment:
we don't want list renderers to take 100% of view port height... This is definitely not the right fix. For instance, go to any form view with an x2many list. `h-100` is probably more appropriate (in future version, we have a scss rule doing that)

Reply:
yes, sorry, the tests were failing with h-100, because the scroll position was not maintained when going back using the breadcrumbs. But adding height 100% specifically on `o_renderer_with_searchpanel` works.